pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -112,6 +113,20 @@ func (c *ClusterController) configureExternalCephCluster(cluster *cluster) error
 		return errors.Wrap(err, "failed to create csi config map")
 	}
 
+	// update the msgr2 flag
+	for _, m := range cluster.ClusterInfo.Monitors {
+		// taking last 4 character will be vaild for both IPv4 Aand IPv6 address
+		// m.Endpoint=10.1.115.104:3300
```

Comment:
Do we need to support external clusters that have ports other than 3300? If so, we'll want to parse this differently since it could be up to 6 chars long I think (e.g., `53300`.

Reply:
There is a [helper](https://github.com/rook/rook/blob/cb44c7b88fb346f3b120e8bda769ac808ec60880/pkg/daemon/ceph/util/util.go#L44) in the utils file to get the port.  That can be used here.

Reply:
Updated